File: library/src/main/java/com/chad/library/adapter/base/loadmore/SimpleLoadMoreView.java
package com.chad.library.adapter.base.loadmore;

import com.chad.library.R;

public final class SimpleLoadMoreView extends LoadMoreView {

  @Override
  public int getLayoutId() {
    return R.layout.quick_view_load_more;
  }

  @Override
  protected int getLoadingViewId() {
    return R.id.load_more_loading_view;
  }

  @Override
  protected int getLoadFailViewId() {
    return R.id.load_more_load_fail_view;
  }

  @Override
  protected int getLoadEndViewId() {
    return R.id.load_more_load_end_view;
  }
}

File: library/src/main/java/com/chad/library/adapter/base/loadmore/SimpleLoadMoreView.kt
package com.chad.library.adapter.base.loadmore

import android.view.View
import android.view.ViewGroup
import com.chad.library.R
import com.chad.library.adapter.base.viewholder.BaseViewHolder
import com.chad.library.adapter.base.util.getItemView

class SimpleLoadMoreView : BaseLoadMoreView() {

    override fun getRootView(parent: ViewGroup): View =
            parent.getItemView(R.layout.brvah_quick_view_load_more)

    override fun getLoadingView(holder: BaseViewHolder): View =
            holder.getView(R.id.load_more_loading_view)

    override fun getLoadComplete(holder: BaseViewHolder): View =
            holder.getView(R.id.load_more_load_complete_view)

    override fun getLoadEndView(holder: BaseViewHolder): View =
            holder.getView(R.id.load_more_load_end_view)

    override fun getLoadFailView(holder: BaseViewHolder): View =
            holder.getView(R.id.load_more_load_fail_view)
}